pkg/kubectl/cmd/get.go

```diff
@@ -262,7 +275,8 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		for ix := range infos {
 			objs[ix], err = infos[ix].Mapping.ConvertToVersion(infos[ix].Object, version.String())
 			if err != nil {
-				return err
+				allErrs = append(allErrs, err)
```

Comment:
Yes, that is correct. `kubectl get` was refactored to use `Visit()` rather than `Infos()`, so that we can aggregate the errors when processing a dir recursively.